UIComponents/UIComponents/ViewController/NavThirdViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  NavThirdViewController.swift
+//  UIComponents
+//
+//  Created by Sagar Ajudiya on 29/05/23.
+//
+
+import UIKit
+
+// MARK: Protocol
+protocol NavThirdViewControllerProtocol: AnyObject {
+    func setPhoneNumber(phoneNumber: String)
+}
```

Comment:
why have we take different protocol for each class for backward data passing, can't we manage this with one protocol and one delegate reference that should be pass to last controller and call method.